force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +181,34 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+        String Grade;
+       if (score >= 90){
+            Grade = 'A';
+           System.debug(Grade);
+            return Grade;
+        }
+        else if (score >= 80 && score < 90) {
+            Grade = 'B';
+            System.debug(Grade);
+            return Grade;
+        }
+        else if (score >= 70 && score < 80) {
+            Grade = 'C';
+            System.debug(Grade);
+            return Grade;
+        }
+        else if (score >= 60 && score < 70) {
```

Comment:
There's not a need to check for the second parameter, you only need the first try it and let me know